tests/test-utils.js

```diff
@@ -0,0 +1,7 @@
+export function getRandomIntInclusive(min, max) {
```

Comment:
Thanks for making this fix, it'll hopefully deflake things! I haven't searched around for vitest / playwright docs yet, but I hope there's something a bit more built-in or recommended floating around. Surely this is a problem anyone running tests encounters. But fine for now.

Reply:
Yeah I couldn't see anything immediately, I was sort of hoping the vite `preview` call would randomly sort out a port, but it just uses a default.